packages/serverless/src/domain/use-case/rollup.ts

```diff
@@ -61,7 +63,7 @@ export async function createRollUp(
     }
   }
 
-  const txId = await enqueueTransaction(
+  const transactionObjectId = await enqueueTransaction(
```

Comment:
Make sure there is no ambiguous variable, `txId` used to reference to `transactionId` or `transactionHash`.

Reply:
done